anypoint-mq/4.x/modules/ROOT/pages/anypoint-mq-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,77 @@
+= Upgrading and Migrating Anypoint MQ Connector Connector to 4.x 
+
+Upgrade Anypoint Connector for Anypoint MQ (Anypoint MQ Connector) to version 4.x.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|3.x |4.x
+|===
+
+== Changes in This Release
+
+Anypoint MQ Connector 4.x includes support for the Connected Apps feature.
+
+The changes included in this version break backward compatibility.
```

Comment:
I think `break in backward compatibility` should be the very first statement while providing upgrade path. 
and details of the features coming in should come after it.

Reply:
Swapped, but not sure what you mean by "while providing upgrade path"